src/components/modal/lagrede-filter/lagrede-filter-utils.tsx

```diff
@@ -0,0 +1,28 @@
+import {FiltervalgModell} from "../../../model-interfaces";
+import {isEmptyArray, isObject} from "formik";
+
+export function lagredeFilterListerErLik(lagretFilter: FiltervalgModell, nyttFilter: FiltervalgModell): boolean {
+    return JSON.stringify(lagretFilter) === JSON.stringify(nyttFilter);
+}
+
+export function erTomtObjekt(objekt): boolean {
+    return Object.values(objekt).length === 0;
```

Comment:
borde det inte vara Object.keys istællet før Object.Values ... ett objekt kan vara {blabla: null, yada:null} och denna funksjonen skulle returnera true

Reply:
but the same issue is with keys...

Reply:
what do you mean that is the same issue with keys?

Reply:
but basically this function should check if object === {}, so it shouldn't have keys or values

Reply:
if an object doest have any keys than it is {} i think

Reply:
nvm